src/api-gateway/resolvers/flag-resolver.ts

```diff
@@ -237,20 +246,24 @@ class UpFlagDetailsArgs {
 
   @Field(() => FallthroughInput, { nullable: true })
   fallthrough: FallthroughInput;
+
+  @Field(() => Boolean, { nullable: true })
+  archived: boolean;
 }
 
 @Resolver()
 export class FlagResolver {
   @Authorized()
   @Query(() => FlagsStatus)
   public async flagsStatus(
-    @Args() { workspaceId, skip, limit }: FlagsArgs,
+    @Args() { workspaceId, skip, limit, archived }: FlagsArgs,
     @Ctx() { model: { flagModel, userWorkspace }, userId }: IContext
   ): Promise<FlagsStatus> {
     await assertWorkspace(userWorkspace, userId, workspaceId);
 
     const query = {
       workspace: workspaceId,
+      archived,
     };
 
     const [total, flags] = await Promise.all([
```

Comment:
should add "archived" in the total count query

Reply:
il seems that estimatedDocumentCount() does not take a query filter ? when i add archived it's not working , or should we use countDocuments({ workspace: workspaceId, archived }) ?

Reply:
seems we have to use "countDocuments" with a query